website/docs/concepts/terminology.md

```diff
@@ -63,7 +63,7 @@ See the [Integration Context](./integration/index.md#integration-context) docume
 
 ## TypeScript Transformer
 
-The Functionless Transformer is a process that runs when you compile your TypeScript code to JavaScript. It re-writes parts of your code to include the [Functionless AST](#abstract-syntax-tree-ast). See [Getting Started](../getting-started.md) for more information on how to configure the transformer.
+The Functionless Transformer is a process that runs when you compile your TypeScript code to JavaScript. It re-writes parts of your code to include the [Functionless AST](#abstract-syntax-tree-ast).
```

Comment:
Hmm, this is not quite true anymore. Swc runs as part of a require hook. It used to work this way

Reply:
This change was just removing the reference since it was no longer correct. Want to take a pass on the rest of the docs in subsequent PRs and keep this one scoped to getting started and get that landed first.